### ISSUE TITLE
Automatic translation add-on removes history

### ISSUE THREAD

**Issue by comradekingu** [WeblateOrg/weblate#6935]
### Describe the issue

https://hosted.weblate.org/addons/remmina/remmina/2752/

with any 100% match for things found in "Weblate translation memory" (1406 components)
removed all accreditation and history for most of the strings.

Don't know why it hasn't done that before, but it is my fault for setting it up like that.
I thought it would only do it for untranslated strings, similar to the option for importing strings.

Is it possible to reset the history to before that happened?

### I already tried

- [X] I've read and searched [the documentation](https://docs.weblate.org/).
- [X] I've searched for similar issues in this repository.

### Steps to reproduce the behavior

1. Go to plugins
2. Turn on auto-translation with 
add as "strings needing work" for "all strings", "machine translation"
3. Select "Weblate translation memory" at "100 points" (perfect matches?)
4. Wait?

### Expected behavior

New translations only for untranslated strings
History kept

### How do you run Weblate?

weblate.org service

**Comment by nijel** [comment]
You probably want to run the automatic translation against untranslated strings only. Running it on all strings will indeed cause what you observe.

**Comment by comradekingu** [comment]
@nijel
Understood. That is what I set it to now.
That leaves the history gone though, which seems like a bug(?)
which is why I am hoping the Remmina project can be reset to the 22.11.2021

**Comment by nijel** [comment]
I don't think that losing history is related to the automatic translation. Most of the strings were introduced by https://gitlab.com/Remmina/Remmina/-/commit/a7b384ba4c85ffded6efeb915b9566583feabea4 and then automatically translated. Can you please share a permalink to a string which should have history and does not have it?

**Comment by comradekingu** [comment]
@nijel I think https://hosted.weblate.org/translate/remmina/remmina/nb_NO/?checksum=bd8f7ef916b48614
Does changing the line placement and then generating the .pot file just do that in any event? If so disregard that part.

https://hosted.weblate.org/translate/remmina/remmina/nb_NO/?checksum=2b85ce03b4ce0e25
is overwritten with the same string, and this string only exists in Remmina since it has the GLADE markup.

If that has to happen for commits to pass, at least those could be removed every now and then?

It is very easy to overwrite the latest translator for everything by uploading a downloaded file.

https://hosted.weblate.org/zen/remmina/remmina/nb_NO/?offset=1&q=+%09changed%3A%3E%3D2010-11-15+AND+NOT+changed_by%3Akingu&sort_by=-priority%2Cposition&checksum=
should not have 800 hits.

This is the worst bug-report on earth, but my general problem is I can't search to
figure out what other people have done, where an actual (changed) translation is the latest (current) entry.

Will return to this once my head clears.
